swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/tagging/ResourceEntityTagger.kt

```diff
@@ -60,10 +60,11 @@ class ResourceEntityTagger(
   }
 
   private fun tagMessage(markedResource: MarkedResource, workConfiguration: WorkConfiguration): String {
-    return markedResource.summaries
+    val summaries = markedResource.summaries
```

Comment:
intellij was telling me there was unreachable code here. Refactoring to be less kotlin-y